### PR TITLE
Azure app configuration

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/appconfiguration/azure-data-appconfiguration/azure/appconfiguration/aio/azure_configuration_client_async.py [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+
+    .. code-block:: python
+
+        from azure.appconfiguration.aio import AzureAppConfigurationClient
+        connection_str = "<my connection string>"
+        async_client = AzureAppConfigurationClient(connection_str)
+    """
+
+    @classmethod
+    def from_connection_string(
```
Comment (johanste) [inline sdk/appconfiguration/azure-data-appconfiguration/azure/appconfiguration/aio/azure_configuration_client_async.py]: `**kwargs` (and pass it through to `__init__`)

File: sdk/appconfiguration/azure-data-appconfiguration/azure/appconfiguration/aio/azure_configuration_client_async.py [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+            cls, connection_string,  # type: str
+    ):
+        return cls(connection_string=connection_string)
+
+    def __init__(self, connection_string):
```
Comment (johanste) [inline sdk/appconfiguration/azure-data-appconfiguration/azure/appconfiguration/aio/azure_configuration_client_async.py]: Take `**kwargs` to initialize policies.

File: sdk/appconfiguration/azure-data-appconfiguration/azure/appconfiguration/aio/azure_configuration_client_async.py [hunk 3]
```diff
@@ -0,0 +1,445 @@
...
+        base_url = "https://" + get_endpoint_from_connection_string(connection_string)
+        program_name = os.path.basename(sys.argv[0]) or "noprogram"
+        self.config = ConfigurationClientConfiguration(
```
Comment (johanste) [inline sdk/appconfiguration/azure-data-appconfiguration/azure/appconfiguration/aio/azure_configuration_client_async.py]: This looks like the old codegen. Configuration objects are gone in the new, correct?

File: sdk/appconfiguration/azure-data-appconfiguration/azure/appconfiguration/aio/azure_configuration_client_async.py [hunk 4]
```diff
@@ -0,0 +1,445 @@
...
+                pass  # do something
+        """
+        labels = escape_and_tolist(labels)
+        keys = escape_and_tolist(keys)
+        return self._impl.list_configuration_settings(
+            label=labels,
+            key=keys,
+            fields=fields,
+            accept_date_time=accept_date_time,
+            headers=kwargs.get("headers"),
```
Comment (johanste) [inline sdk/appconfiguration/azure-data-appconfiguration/azure/appconfiguration/aio/azure_configuration_client_async.py]: Just splat `**kwargs`?

File: sdk/appconfiguration/azure-data-appconfiguration/setup.py [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+with open('HISTORY.md', encoding='utf-8') as f:
+    history = f.read()
+
+exclude_packages = [
+        'tests',
+        'examples',
+        # Exclude packages that will be covered by PEP420 or nspkg
+        'azure',
+    ]
+if sys.version_info[0] < 3 or (sys.version_info[0] == 3 and sys.version_info[1] < 5):
```
Comment (johanste) [inline sdk/appconfiguration/azure-data-appconfiguration/setup.py]: I'd humbly suggest `if sys.version_info < (3, 5):`...

**Pull request closed without merging**